packages/renderer/src/lib/feedback/feedbackForms/DevelopersFeedback.svelte

```diff
@@ -25,6 +28,8 @@ $: hasFeedback =
   (tellUsWhyFeedback && tellUsWhyFeedback.trim().length > 4) ||
   (contactInformation && contactInformation.trim().length > 4);
 
+$: dispatch('propsAreEmpty', !(smileyRating || tellUsWhyFeedback || contactInformation));
```

Comment:
avoid using dispatch, either migrate the component to svelte 5 or use a callback props